templates/team.html

```diff
@@ -24,7 +24,7 @@ <h2 class="text-size-2">
                 <div class="col-md-10 col-12 offset-md-1 text-center">
                     <p class="subtext-size-2 margin-top-10">
                         We're a fully remote software development and ops team working from<br/>
-                        Montreal, California, Medellin, and NYC.<br><br>
+                        Montreal, California, Medellín, and NYC.<br><br>
```

Comment:
If we're adding accents we should add all of them ;)
```suggestion
                        Montréal, California, Medellín, and NYC.<br><br>
```